content/intro/pricing/pricing-model.md

```diff
@@ -8,45 +8,25 @@ weight: 10
+Find out more about each component via our [pricing terminology page]({{< relref "intro/terminology/pricing-terminology.md" >}}) or see the [dedicated page with our rates]({{< relref "intro/pricing/rates.md" >}}). 
 
-## Resource usage
-
-To support tenant applications and managed service instances, cloud.gov allocates resources (like compute nodes and memory) from an underlying IaaS. What we charge for this component covers the costs for the resources that cloud.gov allocates to hosted systems and teams. It also includes a small margin that covers our labor in managing those resources.
-
-Resource usage fees on-demand are understood to be “severable” as they can be controlled in a self-service manner by provisioning more or fewer apps, and therefore must be treated accordingly.
-
-Agencies can purchase reserved capacity via 18F as well, which is understood to be “non-severable”.
-
-## Managed services
-
-Each “managed service” (see our definition) incurs some resource usage and platform support overhead. The managed service fees ensure we can cover our costs to provide them to you. The fee varies among services and plans.
-
-In many cases, the pricing for managed services is not yet available. This is because we are measuring their initial usage to determine what we will need to charge.
+---
+Our mission is to catapult government agencies past mere cloud 'adoption', to the point where they routinely deliver services to the public as fast as they can be developed, all while applying the best practices in security and compliance with minimal effort. 
 
-We do not yet have a good picture of how much we should charge to cover the cost of these services. We are doing more analysis of the costs we’re observing (both the cost of the IaaS layer and the cost of our support). We aim to offer explicit prices for all these by the end of March 2016.
+Comparing cloud.gov's pricing model to those private Platform as a Service (PaaS) providers is extremely difficult, as it is not an apples to apples comparison. If you attempt to do a price comparison, be sure to include the up-front and opportunity costs of your procurement, operations, security, and compliance teams in obtaining a solution from scratch. It may be difficult to obtain these costs, especially if they are hidden within various working captial funds in your agency. Consider the labor and time to be saved by having a proven, ready-to-ATO infrastructure in your teams' hands from Day One, via a relatively simple Inter-agency Agreement (IAA). Cloud.gov's resource usage is already priced competitive with the private market, and is the superior option in reducing compliance friction to access the cloud. 
 
-## À la carte items
+It is critical to obtain all this data in order to perform a true cost of ownership comparison. 18F stands ready to help you build an accurate cost model.
```

Comment:
:+1: 

Maybe sufficient to say "via our [Consulting](https://18f.gsa.gov/consulting/) branch"?